Magento2/Tests/Legacy/WidgetXMLUnitTest.1.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<widgets>
+    <widget type="/Deprecated">
+        <label translate="true">Catalog New Products List</label>
+        <description translate="true">List of Products that are set as New</description>
```

Comment:
Include another file like this, but having the `translate` attribute in a new line to check also badly formatted XML files

Reply:
No needed in a specific widget sniff. Just removed that check.